crates/rnote-engine/src/selectioncollision.rs

```diff
@@ -0,0 +1,23 @@
+#[derive(Clone, Debug, Default)]
+#[cfg_attr(feature = "clap-derive", derive(clap::ValueEnum))]
+pub enum SelectionCollision {
+    #[default]
+    /// All strokes completely inside the area
+    Contains,
+    /// All Strokes intersecting with the area
+    Intersects,
+}
+
+#[cfg(feature = "clap-derive")]
```

Comment:
remove this condition, implementing `Display` does not need to be conditional to the clap-derive feature